ddl/placement/utils.go

```diff
@@ -69,3 +70,8 @@ func CheckLabelConstraints(labels []string) ([]LabelConstraint, error) {
 	}
 	return constraints, nil
 }
+
+// GroupID accepts a tableID or whatever integer, and encode the integer into a valid GroupID for PD.
+func GroupID(id int64) string {
+	return fmt.Sprintf("TIDB_DDL_%d", id)
```

Comment:
```suggestion
	return fmt.Sprintf("%s%d", RuleDefaultGroupID, id)
```

Reply:
That variable is going to be removed in the next three PRs.

Reply:
OK.